src/vs/server/node/remoteLanguagePacks.ts

```diff
@@ -44,3 +50,39 @@ export namespace InternalNLSConfiguration {
 		return candidate && typeof candidate._languagePackId === 'string';
 	}
 }
+
+export const getLocaleFromConfig = async (argvResource: string): Promise<string> => {
```

Comment:
May be able to use this function: https://github.com/microsoft/vscode/blob/f3a9245af988874dd1bae7281d904ff6133304c6/src/main.js#L585

Reply:
Actually...not 1:1 and can't import from JS file I don't think (well I don't see any other files using this since it's not exported) so going to leave as is.

Reply:
Similar function/logic as well: https://github.com/microsoft/vscode/blob/f3a9245af988874dd1bae7281d904ff6133304c6/src/main.js#L233-L256